### PR TITLE
Include optimized inline svg icons for amp-social-share

### PR DESCRIPTION
**Pull request by paul-matthews**
Including Optimized Inline SVG icons third_party code for amp-social-share ( #1201 )

* Includes raw files
* Described the third_party lib in README.amp
* Added local changes in amp-social-share-svgs.css

### CODE REVIEW

File: third_party/optimized-svg-icons/README.amp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+URL: http://codepen.io/anon/pen/KVZNGp
+License: The MIT License
+License File: https://gist.githubusercontent.com/anonymous/920164b2cafcdade77a2/raw/97801f67daefc7cde3523d9c35ef73f6a43759f5/script.js
```
Comment (erwinmombay) [inline third_party/optimized-svg-icons/README.amp]: required to have a `LICENSE` file in this folder too

Comment (paul-matthews) [inline third_party/optimized-svg-icons/README.amp]: Done.

**Comment by erwinmombay** [comment]
@paul-matthews can you rebase and push to get a green build? this LGTM

**Comment by erwinmombay** [comment]
@cramforce can you take a quick look and see if we missed anything?

**Comment by erwinmombay** [comment]
@cramforce ping. will merge this afternoon if you dont see any issues.

**Comment by cramforce** [comment]
LGTM

**Pull request merged**